docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1038,6 +1038,23 @@ How to **investigate**:
   1. The alert fired because of a bug in Mimir: fix it.
   1. The alert fired because of a bug or edge case in the continuous test tool, causing a false positive: fix it.
 
+### RolloutOperatorNotReconciling
+
+This alert fires if the [`rollout-operator`](https://github.com/grafana/rollout-operator) is not successfully reconciling in a namespace.
+
+How it **works**:
+
+- The operator coordinates the rollout of pods between different StatefulSets within a specific namespace and is used to manage multi-zone deployments
```

Comment:
I think saying "Rollout Operator" instead of operator would be more clear.

Reply:
Done